Google/resource-manager/project/README.md

```diff
@@ -26,7 +28,59 @@ The `components.specs` block contains maps of project configuration, with the fo
 | `auto_create_network` | boolean |  false   | automatically create a default network in the Google project                                            |                   none                    |
 | `project_iam`         |  list   |  false   | List of IAM role bindings used to create IAM policy for the project (see details [below](#project-iam)) |                   none                    |
 
-### Project IAM
+### Parent configuration  
+The parent of the project must be either a folder or an organization, and can be configured by the `parent_folder`/`parent_org` variables, as well as in the MCCF file.  
+If these values are set in multiple places, the order of priority is:
+1. Variables set in module block. The `parent_folder` or `parent_org` variable will take priority over any MCCF configuration,
```

Comment:
Why are we referring to module block here. I had to stop and think what this meant, so someone else would certainly be confused. This would be the values set in vars, right?